blocks/share-bar-block/features/share-bar/default.test.jsx

```diff
@@ -26,44 +22,49 @@ describe('When the share bar is shown', () => {
         },
       })),
     }));
+*/
 
-    getProperties.mockImplementation(() => ({
-      websiteDomain: 'https://www.thesun.com/',
-      websiteName: 'The Sun',
-    }));
-
-    const wrapper = mount(<ShareBar />);
+describe('When the share bar is shown', () => {
+  const websiteDomain = 'https://www.thesun.com/';
+  const websiteName = 'The Sun';
+  const websiteUrl = '/2019/07/15/global-kitchen-sink-article/';
+  const headlineString = 'sample headline';
```

Comment:
global variables basically reused throughout the tests, didn't need to repeat. only customFields is really changed

Reply:
import { ShareBar } from "./features/share-bar/default";
import data from "./helper";
import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";

<Meta title="Share Bar" component={ShareBar} />

# Share Bar

The `ShareBar` block is written as a functional component. Users can choose out of five options (email, facebook, pinterest, linkedin, twitter) which selection of share buttons will be displayed on articles. Clicking on each button will open up a new window.

## Use

To import it into a block:

```
import ShareBar from '@wpmedia/share-bar-block';

...
    <ShareBar
        useInjectedFusionContext={{
            customFields: {
                email: false,
                facebook: true,
                pinterest: false,
                twitter: false,
                linkedIn: false,
            },
            globalContent: {
                label: {
                basic: {
                    display: true,
                    url: 'http://google.com/',
                    text: 'Overline Text Root',
                },
                },
            },
        }}
    />
...

```

## Props

<Props of={ShareBar} />

## Stories

** Custom ShareBar **

<Preview>
  <Story name="Custom ShareBar">
    <ShareBar {...data} />
  </Story>
</Preview>

Reply:
@beltrancaliz if you have time, I was struggling with using mdx with our new setup without the top-level object import: 

```

import data from './helper'
** Custom ShareBar **

<Preview>
  <Story name="Custom ShareBar">
    <ShareBar {...data} />
  </Story>
</Preview>

Reply:
```
import { boolean, text } from '@storybook/addon-knobs';

const data = () => {
  const headlineString = text('headlineString', 'Man Bites Dog');
  const websiteDomain = text('websiteDomain', 'https://www.thesun.com/');
  const websiteUrl = text('websiteUrl', '/2019/07/15/global-kitchen-sink-article/');
  const websiteName = text('websiteName', 'The Sun');

  const emailInput = boolean('email', true);
  const facebookInput = boolean('facebook', true);
  const pinterestInput = boolean('pinterest', true);
  const twitterInput = boolean('twitter', true);
  const linkedInInput = boolean('linkedIn', true);

  const customFields = {
    email: emailInput,
    facebook: facebookInput,
    pinterest: pinterestInput,
    twitter: twitterInput,
    linkedIn: linkedInInput,
  };

  return ({
    customFields,
    headlineString,
    websiteDomain,
    websiteUrl,
    websiteName,
  });
};